### ISSUE TITLE
visualize_topics_over_time normalize_frequency parameter not working

### ISSUE THREAD

**Issue by doubianimehdi** [MaartenGr/BERTopic#213]
Hi,

When I try to set the normalize_frequency parameter for the visualize_topics_over_time function I have the following message : 
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
~\AppData\Local\Temp/ipykernel_6188/576566513.py in <module>
----> 1 topic_model.visualize_topics_over_time(topics_over_time, top_n_topics=50, normalize_frequency=True, width=900, height=500)

TypeError: visualize_topics_over_time() got an unexpected keyword argument 'normalize_frequency'

But I can see that parameter in the code .... 
What Am I missing ? 
Thanks !

**Comment by MaartenGr** [comment]
Although the newest version of BERTopic has this new parameter, I forgot to actually put it in the `_bertopic.py` reference. No worries, it will be fixed in the next release among some other fixes. 

For now, you can use it as follows:

```python
from bertopic import plotting
plotting.visualize_topics_over_time(topic_model, topics_over_time, top_n_topics=20, normalize_frequency=True)
```

**Comment by MaartenGr** [comment]
A fix for this issue has been released (v0.9.1). To use this fix, update BERTopic with: `pip install --upgrade bertopic`. If there any other issues you are experiencing, please let me know!
